derivation.nix

```diff
@@ -0,0 +1,24 @@
+{stdenv, buildGoModule}:
+
+let
+in buildGoModule rec {
```

Comment:
Does this pattern mean that someone using this `derivation.nix` can easily inject custom attributes to the set?

Reply:
No, this is a custom builder provided by nixpkgs for builiding go programs managed with go modules. I tried to follow the [official nixpkgs documentation](https://nixos.org/nixpkgs/manual/#ssec-go-modules), which suggest using `buildGomodule`. But yes, you could create an [overlay](https://nixos.wiki/wiki/Overlays) for this if you wanted to 'inject' things.

Reply:
Nah that's merely just a leftover. I used to have stuff in there which disappeared over time. I will remove this.